provisioning/resources/ui/index.html

```diff
@@ -1,43 +1,146 @@
+            <div class="description">
+                Snowplow Mini is, in essence, the Snowplow real time pipeline containerized within a single box. In place of Kinesis Streams we are using NSQ and instead of distributing all of the applications across Autoscaling Groups they are deployed onto a single instance.
+            </div>
+
+            <section class="section">
+                <h2>1. Sending events</h2>
+                <p>You can send events into Snowplow Mini automatically from the <a href="#example-events">Example events</a> page. Simply go to that page and click the sample event buttons.</p>
+                <p>Alternatively, you can setup any of the Snowplow trackers to send data to this endpoint: <span id="collector-endpoint"></span></p>
+            </section>
+
+            <section class="section">
+                <h2>2. Viewing the events</h2>
+                <p>You can view the events that have been sent to Opensearch in the <a href="/kibana">Opensearch Dashboards</a>.</p>
+                <p>You can also submit queries directly to the <a href="/elasticsearch">Opensearch endpoint</a>.</p>
+            </section>
+
+            <section class="section">
+                <h2>3. Understanding how Snowplow Mini works</h2>
+
+                <div class="quicklinks">
+                    <h3>Quicklinks:</h3>
+                    <ul class="link-list">
+                        <li><a href="https://docs.snowplowanalytics.com/docs/understanding-your-pipeline/what-is-snowplow-mini">What is Snowplow Mini?</a></li>
+                        <li><a href="https://docs.snowplowanalytics.com/docs/pipeline-components-and-applications/snowplow-mini/usage-guide/">Usage guide</a></li>
+                        <li><a href="https://docs.snowplowanalytics.com/docs/pipeline-components-and-applications/snowplow-mini/control-plane-api/">Control Plane API</a></li>
+                        <li>Link to <a href="https://github.com/snowplow/snowplow-mini">Snowplow Mini</a> repository</li>
+                        <li>Collector endpoint <a href="/" id="collector-link"></a></li>
+                        <li>Metrics endpoint <a href="/metrics" id="metrics-link"></a></li>
+                    </ul>
+                </div>
+
+                <div class="software-stack">
+                    <h3>The software stack installed:</h3>
+                    <ul class="software-list">
+                        <li>Snowplow Mini 0.23.2</li>
+                        <li>Snowplow Collector NSQ 3.4.0</li>
+                        <li>Snowplow Enrich NSQ 5.4.0</li>
+                        <li>Snowplow Elasticsearch Loader 2.1.3</li>
+                        <li>Snowplow Iglu Server 0.14.0</li>
+                        <li>Postgres 15.1</li>
+                        <li>NSQ v1.3.0</li>
+                        <li>Opensearch 2.19.1</li>
+                        <li>Opensearch Dashboards 2.19.1</li>
+                        <li>Cadvisor 0.52.1</li>
```

Comment:
Do we need to keep these manually updated? I guess we could easily forget about it? Can we just remind to an external page (doc or somewhere else)? Forget it, if it was already that way before